packages/E2ETest/windows/ReactUWPTestApp/Assets/TreeDump/ImageWithBorder.treedump

```diff
@@ -0,0 +1,16 @@
+[Windows.UI.Xaml.Controls.Border]
+    Padding=0,0,0,0
```

Comment:
Possible just make the extension name .treedump as .txt. customer doesn't need to setup a tool to open .treedump.

Reply:
Will do.

---
In reply to: [356300132](https://github.com/microsoft/react-native-windows/pull/3754#discussion_r356300132) [](ancestors = 356300132)